src/main/java/com/github/vkorobkov/jfixturescmd/utils/ExceptionHandler.java

```diff
@@ -0,0 +1,13 @@
+package com.github.vkorobkov.jfixturescmd.utils;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public final class ExceptionHandler {
+    private ExceptionHandler() {
+    }
+
+    public static void handleException(Exception exception) {
+        log.error(exception.getMessage());
```

Comment:
Let's check that:
1) Exception is `LoaderException`
  1a) If the root cause in FileMissing - print a specific message
  ab) for other root causes - print a generic message
2) Print a generic message/stacktrace for other exceptions